azure/template.json

```diff
@@ -291,12 +373,15 @@
                 }
             },
             "dependsOn": [
-                "function-app-service-plan"
+                "function-app-service-plan",
+                "function-app-insights",
```

Comment:
Implicit dependency created for app-insights and subnet by use of reference function